src/Document/elements.ts

```diff
@@ -0,0 +1,137 @@
+import Element from './Element';
+import UnknownElement from './UnknownElement';
+import RenderedElement from './RenderedElement';
+import PathElement from './PathElement';
+import SVGElement from './SVGElement';
+import RectElement from './RectElement';
+import CircleElement from './CircleElement';
+import EllipseElement from './EllipseElement';
+import LineElement from './LineElement';
+import PolylineElement from './PolylineElement';
+import PolygonElement from './PolygonElement';
+import PatternElement from './PatternElement';
+import MarkerElement from './MarkerElement';
+import DefsElement from './DefsElement';
+import GradientElement from './GradientElement';
+import LinearGradientElement from './LinearGradientElement';
+import RadialGradientElement from './RadialGradientElement';
+import StopElement from './StopElement';
+import AnimateElement from './AnimateElement';
+import AnimateColorElement from './AnimateColorElement';
+import AnimateTransformElement from './AnimateTransformElement';
+import FontElement from './FontElement';
+import FontFaceElement from './FontFaceElement';
+import MissingGlyphElement from './MissingGlyphElement';
+import GlyphElement from './GlyphElement';
+import TextElement from './TextElement';
+import TSpanElement from './TSpanElement';
+import TRefElement from './TRefElement';
+import AElement from './AElement';
+import TextPathElement from './TextPathElement';
+import ImageElement from './ImageElement';
+import GElement from './GElement';
+import SymbolElement from './SymbolElement';
+import StyleElement from './StyleElement';
+import UseElement from './UseElement';
+import MaskElement from './MaskElement';
+import ClipPathElement from './ClipPathElement';
+import FilterElement from './FilterElement';
+import FeDropShadowElement from './FeDropShadowElement';
+import FeMorphologyElement from './FeMorphologyElement';
+import FeCompositeElement from './FeCompositeElement';
+import FeColorMatrixElement from './FeColorMatrixElement';
+import FeGaussianBlurElement from './FeGaussianBlurElement';
+import TitleElement from './TitleElement';
+import DescElement from './DescElement';
+
+export default {
+	'svg':              SVGElement,
```

Comment:
can we make an array of elements and loop over them and ask each for it's `.type`? that way we're not duplicating strings

Reply:
`.type` is not static property

Reply:
is there a reason why not? something from my existing code?